### PR TITLE
Check joints

### PR DESCRIPTION
**Pull request by maroqijalil**

### CODE REVIEW

File: src/tachimawari_dynamixel/cm740.cpp [hunk 1]
```diff
@@ -510,7 +500,7 @@ bool CM740::dxl_power_on()
     }
 
     write_word(ID_CM, CM_LED_HEAD_L, make_color(255, 128, 0), 0);
-    sleep(300);  // about 300msec
+    std::this_thread::sleep_for(std::chrono::milliseconds(300));  // about 300msec
```
Comment (threeal) [inline src/tachimawari_dynamixel/cm740.cpp]: use literals (`300ms`), use `using namespace std::chrono_literals` first.

File: src/tachimawari_dynamixel/cm740.cpp [hunk 2]
```diff
@@ -837,38 +831,6 @@ bool CM740::open_port()
   return true;
 }
 
-bool CM740::set_baud(int baud)
```
Comment (threeal) [inline src/tachimawari_dynamixel/cm740.cpp:840]: Just asking, but why is it removed? and changed into a simple property setter?

**Review by threeal — CHANGES_REQUESTED** [review]

**Review by threeal — APPROVED** [review]
Looks good to me.

**Review by finesaaa — APPROVED** [review]
Good 💯

**Pull request merged**